sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified_comments.json

```diff
@@ -2866,24 +2864,24 @@
       }
     },
     "CloudToDeviceMethod": {
-      "description": "Parameters to execute a direct method on the device",
+      "description": "Parameters to execute a direct method on the device.",
       "type": "object",
       "properties": {
         "methodName": {
-          "description": "Method to run",
+          "description": "Name of the method to be executed.",
           "type": "string"
         },
         "payload": {
-          "description": "Payload",
+          "description": "The JSON-formatted direct method payload, up to 128kb in size.",
           "type": "object"
         },
         "responseTimeoutInSeconds": {
-          "description": "Timeout for the method invocation to return a response",
+          "description": "Time (in seconds) that the service waits for the method invocation to return a response. It defaults to 30 seconds. Minimum allowed value is 5 seconds, maximum allowed value is 300 seconds.",
```

Comment:
Maybe reword? Default is 30 seconds; Minimum allowed value is 5 seconds, Maximum allowed value is 300 seconds.